Automattic/wp-calypso
# Domains reskin: Improve mobile view - take 2

### Changes proposed in this Pull Request
All proposed changes from #52905, except for the search filters, are implemented.

### Testing instructions
See gnYBUOBthTMckJRJr3pZmk-fi-2598%3A1 for reference.

1. Check out this branch and `yarn && yarn start`
2. Go to /start/domains?flags=signup/reskin
3. Switch to a mobile viewport.
4. Search for a domain.
5. Ensure that:
    - [ ] Select button is below the copy and is full-width.
    - [ ] Font size for domain suggestions is 16px.
    - [ ] The 'Show more results' link is now a button (same style as the 'Select' buttons in domain suggestions).
    - [ ] The background of the first 2 domain suggestions is #ffffff.
    - [ ]  The border color of the active search bar is #646970.
    - [ ]  'Already own a domain?' and 'Get a free plan' information containers are moved below the search results.

#### Regression testing
1. Go to /start/domains
2. Switch to a mobile viewport.
3. Search for a domain.
4. Ensure that (see screenshot):
    - [ ] "Get a free one-year domain registration with any paid annual plan" banner is right between the search bar and domain suggestions.
    - [ ] There is no gap between the domain suggestions and the "show more results" button.
5. Repeat steps 3 & 4 for both tablet & desktop viewports.
 
<img width="809" alt="122588480-d3e2d800-d067-11eb-8505-a89b9097466b" src="https://user-images.githubusercontent.com/12104589/122760271-37a11700-d29b-11eb-8330-01b8c2c74ada.png">


Fixes #52905
